doc/admin-guide/files/records.config.en.rst

```diff
@@ -3613,6 +3613,14 @@ OCSP Stapling Configuration
 
    Update period (in seconds) for stapling caches.
 
+.. ts:cv:: CONFIG proxy.config.ssl.ocsp.response.path STRING NULL
+
+   The location of the prefetched OCSP stapling responses. Change this
```

Comment:
Would it be more clear to say this is a directory? For instance, "The directory path of the ..."